src/shared-components/button/index.tsx

```diff
@@ -63,99 +63,83 @@ type ButtonProps = {
  *
  * We should generally try to use the default button color when possible. Only for special cases should we need to use a different button color.
  */
-export class Button extends React.Component<ButtonProps> {
-  static Container = Container;
+export const Button = ({
+  buttonColor = COLORS.primary,
+  buttonType = 'primary',
+  children,
+  disabled = false,
+  icon = null,
+  isFullWidth = false,
+  isLoading = false,
+  loading = undefined,
+  onClick = () => undefined,
```

Comment:
[Nit] This is out of the scope for this PR but I don't think a Button without an `onClick` makes sense. If we want I think it's within our purview to remove this for improved type sensibility.

Reply:
So You are saying to make onClick required? I changed to required